src/routes/+layout.svelte

```diff
@@ -255,7 +255,7 @@
 		<!-- Middle -->
 		{#if innerWidth >= tailwindXsScreen}
 			<button
-				class="absolute bottom-0 left-0 right-0 text-center"
+				class="absolute bottom-0 left-0 right-0 text-center w-fit mx-auto"
```

Comment:
Great!! I was thinking about wrapping this in a `<div>` but this is even better, congrats!

(Same remark about Prettier as below)